tests/sdk/nodejs/nodejs_test.go

```diff
@@ -449,10 +452,24 @@ func TestCRDs(t *testing.T) {
 			},
 		},
 	})
+	test, _ = testClusters.WrapProviderTestOptions(test)
 	integration.ProgramTest(t, &test)
 }
 
 func TestPod(t *testing.T) {
+	getCondition := func(conditions []any, conditionType string) map[string]any {
```

Comment:
I suppose that `FindStatusCondition` wasn't useful here?
https://github.com/kubernetes/apimachinery/blob/v0.29.2/pkg/api/meta/conditions.go#L91

Reply:
The helper function is for typed conditions, whereas the object we are working with is unstructured.